tests/test_freqtradebot.py

```diff
@@ -3585,34 +3582,43 @@ def test_get_real_amount_no_currency_in_fee(default_conf, trades_for_order, buy_
     )
     freqtrade = get_patched_freqtradebot(mocker, default_conf)
 
-    # Amount does not change
-    assert freqtrade.get_real_amount(trade, limit_buy_order) == amount
+    if not use_ticker_rate:
+        mocker.patch('freqtrade.exchange.Exchange.fetch_ticker', side_effect=ExchangeError)
 
+    caplog.clear()
+    # Amount does not change
+    assert freqtrade.get_real_amount(trade, buy_order) == amount - fee_reduction_amount
+
+    if expected_log:
+        assert log_has(expected_log, caplog)
+
+
+@pytest.mark.parametrize(
+    'stake_currency, fee_cost, fee_currency, fee_reduction_amount, expected_fee, expected_log', [
+        ("BTC", None, None, 0.001, 0.001, (
```

Comment:
What's the point of stake_currency here (as parameter, i understand it might need to be set as 

BTC seems wrong (considering trades are for LTC/ETH) - and changing it to ETH doesn't seem to have any effect (i also struggle to see where we set stake_currency to BTC in the old tests ...)

Reply:
That's what is in the `default_conf` and since the old `test_get_real_amount_multi2` test was setting it to ETH, being more explicit in the consolidation seemed reasonable.

Reply:
> That's what is in the `default_conf` and since the old `test_get_real_amount_multi2` test was setting it to ETH, being more explicit in the consolidation seemed reasonable.

I would remove it